### PR TITLE
Rnn cleanup

### PR DESCRIPTION
**Pull request by robertleeplummerjr**
A GIF or MEME to give some spice of the internet

## Motivation and Context

[issue](https://github.com/BrainJS/brain.js/issues/364)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Author's Checklist:

- [ ] My code focuses on the main motivation and avoids scope creep.
- [ ] My code passes current tests and adds new tests where possible.
- [ ] My code is [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).
- [ ] I have updated the documentation as needed.

## Reviewer's Checklist:
- [ ] I kept my comments to the author positive, specific, and productive.
- [ ] I tested the code and didn't find any new problems.
- [ ] I think the motivation is good for the project.
- [ ] I think the code works to satisfies the motivation.

### CODE REVIEW

File: src/layer/add.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ class Add extends Operator {
     this.inputLayer1.deltas = clone(this.deltas);
     this.inputLayer2.deltas = clone(this.deltas);
   }
+
+  learn() {}
```
Comment (mubaidr) [inline src/layer/add.js:55]: Empty? is it expected?

Comment (robertleeplummerjr) [reply src/layer/add.js:55]: It's just abstract is all

File: src/recurrent/matrix/equation.js [hunk 2]
```diff
@@ -229,9 +229,11 @@ class Equation {
     let iBackpropagate = 0;
     this.states.push({
       forwardFn() {
+        console.log(m);
```
Comment (mubaidr) [inline src/recurrent/matrix/equation.js]: Console copmments

File: src/recurrent/matrix/equation.js [hunk 3]
```diff
@@ -229,9 +229,11 @@ class Equation {
...
         iForward++;
       },
       backpropagationFn() {
+        console.log(m);
```
Comment (mubaidr) [inline src/recurrent/matrix/equation.js]: console comments

File: src/recurrent/matrix/equation.js (hunk 2 above)
Comment (robertleeplummerjr) [reply src/recurrent/matrix/equation.js]: Fixed

File: src/recurrent/matrix/equation.js (hunk 3 above)
Comment (robertleeplummerjr) [reply src/recurrent/matrix/equation.js]: Fixed

File: src/layer/add.js (hunk 1 above)
Comment (robertleeplummerjr) [reply src/layer/add.js:55]: pushed description

**Review by mubaidr — APPROVED** [review]

**Pull request merged**